README.md

```diff
@@ -9,6 +9,14 @@ The Node.js project uses an [open governance model](./GOVERNANCE.md). The
 
 **This project has a [Code of Conduct][].**
 
+In addition to the code of conduct requirements, contributors to the
+Node.js project are expected to act in a collaborative manner to move
+the project forward. We encourage the constructive exchange of contrary
+opinions and compromise. Contributors who repeatedly act in ways that
+discourage, exhaust, or otherwise negatively affect other contributors
+will be limited in the roles they are allowed to hold or blocked
+from the project completely.
```

Comment:
```suggestion
opinions and compromise. The TSC reserves the right to limit or block
contributors who repeatedly act in ways that discourage, exhaust, or
otherwise negatively affect other participants.
```

Reply:
I like this suggested change, will incorporate after giving others a bit of time to comment one way or the other.

Reply:
We should probably link to the definition of TSC in `./GOVERNANCE.md#technical-steering-committee`, as the reader may not be aware of what the TSC is.